libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs

```diff
@@ -22,6 +23,7 @@ namespace Microsoft.Bot.Builder.Dialogs
     public class SkillDialog : Dialog
     {
         private const string DeliverModeStateKey = "deliverymode";
+        private const string SsoConnectionNameKey = "SSOConnectionName";
```

Comment:
Can we namespace this key?  Since SkillDialog has already been shipped we can't guarantee that this key is available for our use.